### ISSUE TITLE
[bug]backquote with count(*) problem 

### ISSUE THREAD

**Issue by ttamatt** [orientechnologies/orientdb#8544]
## FAILED SQL
```
SELECT name as `name` , count(*) FROM V WHERE name IN ['Jack', 'Rose'] GROUP BY name LIMIT -1
```
## SUCCESSFUL SQL  
```
SELECT name as `name`   FROM V WHERE name IN ['Jack', 'Rose'] GROUP BY name LIMIT -1
```
## Expected behavior  

 | PROPERTIES    |
|---------------|
| Jack |
|Rose |

## Actual behavior  
>Empty data

When I tried to use backquote with count(*), it doesn't return anything, but using `name` alone is okay, may you find the reason please?

Thanks,
Matt

**Comment by luigidellaquila** [comment]
Hi @ttamatt 

I just pushed a fix, it will be released with 3.0.8

Thanks

Luigi

**Comment by ttamatt** [comment]
Thanks！
